src/Repository/UserRepository.php

```diff
@@ -105,7 +105,7 @@ public function findByFilters(array $formData, bool $onlyIds = false): array
                 $qb->setParameter(sprintf('skill%d', $key), $skill);
             }
 
-            $qb->andWhere($qb->expr()->orX(...$skillsQueries));
+            $qb->andWhere($qb->expr()->andX(...$skillsQueries));
```

Comment:
This change will break everything on the app, as this function is used in many different situations.